espflash/src/cli/mod.rs

```diff
@@ -59,6 +60,10 @@ pub struct FlashOpts {
     /// Open a serial monitor after flashing
     #[clap(long)]
     pub monitor: bool,
+    /// Erase the OTADATA partition
+    /// This is useful when using multiple OTA partitions and still wanting to be able to reflash via espflash
+    #[clap(long)]
+    pub erase_otadata: bool,
```

Comment:
Can't you model the dependency using `requires = "use-stub"`?

Reply:
Thanks for the idea, used in https://github.com/esp-rs/espflash/pull/273